src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -139,18 +140,36 @@ julia> vertices(square)
  [-1, 1]
  [1, 1]
 
-julia> nc = normal_cone(square, 1)
+julia> nc = normal_cone(square, [1])
```

Comment:
```suggestion
julia> nc = normal_cone(square, 1)
```
Please do not change the function signature as this would break the API. Instead add a new signature.